tip-wlan/charts/wlan-prov-service/templates/configmap.yaml

```diff
@@ -4,16 +4,16 @@ metadata:
   name: {{ include "common.fullname" . }}-log-config
   namespace: {{ include "common.namespace" . }}
 data:
-{{ tpl (.Files.Glob "resources/config/*").AsConfig . | indent 2 }}
+{{ (.Files.Glob "resources/config/logback.xml").AsConfig | indent 2 }}
   datasource.properties: |-
     singleDataSource.url=jdbc:postgresql://{{- include "postgresql.service" . -}}:5432/prov_db
-    singleDataSource.username={{ .Values.creds.postgres.singleDataSourceUsername }}
-    singleDataSource.password={{ .Values.creds.postgres.singleDataSourcePassword }}
+    singleDataSource.username={{ .Values.global.postgres.singleDataSource.username }}
```

Comment:
When we use global values, we should probably provide an option to override them locally. A pattern used across many Helm charts goes something like: `{{ .Values.global.postgres.singleDataSource.username | default .Values.postgres.singleDataSource.username }}`

Reply:
does it make any sense though? whats the point of overriding them if they are defined in the global. if you override it in local to any value except the one in global it would not work

Reply:
The only use case I could think of where this is necessary is if we allow to use existing and different e.g. PSQL databases for each service. I don't think that that's a requirement for now so I'd like to prevent to make the chart even more complex for now.

Reply:
Ok, agree with you guys on this one.

Reply:
i dont think we allow to specify different psql hostname anyway